tools/smxtools/smxdasm/Headers.cs

```diff
@@ -140,9 +140,13 @@ public static FileHeader From(BinaryReader rd)
             switch (header.Compression)
             {
             case CompressionType.None:
+            {
                 // This case is easy... we can just read the rest of the file.
                 rd.Read(header.Data, Size, header.ImageSize - Size);
+                var new_stream = new MemoryStream(header.Data, Size, header.ImageSize - Size);
```

Comment:
This is redundant since that exact code was moved below the switch already.
https://github.com/alliedmodders/sourcepawn/commit/eb4a2e00aba16be70f79d90c60cfa9d80a808a57

Reply:
Thanks, I missed that.